File: Two Furthest Houses With Different Colors.cpp
class Solution {
public:
    int maxDistance(vector<int>& colors) {
        int maxx=0;
        for(int i=0;i<colors.size();i++)
        {
            for(int j=i;j<colors.size();j++)
            {
                if(colors[i]!=colors[j])
                    maxx=max(j-i, maxx);
            }
        }
        return maxx;
    }

File: LeetCode-Programs/Two Furthest Houses With Different Colors.py
class Solution:
    def maxDistance(self, colors: List[int]) -> int:
        maxx=0
        for i in range(len(colors)):
            for j in range(i,len(colors)):
                if colors[i]!=colors[j]:
                    maxx=max(j-i, maxx)
        return maxx
            
